src/main/java/com/wolox/training/exceptions/ErrorDescription.java

```diff
@@ -0,0 +1,22 @@
+package com.wolox.training.exceptions;
+
+public enum ErrorDescription {
+
+    BOOK_NOT_FOUND("Book not found.", 1);
+
+    private String reason;
+    private Integer code;
```

Comment:
I think that in this case, the fields have to be final, because you'll never set a new value.